fcc-test-suite-template-master/index.html

```diff
@@ -39,11 +39,14 @@ <h1><center>Here's a time line of Dr. César's life:</center></h1>
             Works in the Civilian Conservation Corps, helping starving
             Americans. "I saw how food changed them", he said. "All of this left
             scars on me."
+            <strong></strong>
```

Comment:
please remove  unimportant tags and attributes, because it makes you code un clean and hard to read 🙌